homeassistant/components/plugwise/climate.py

```diff
@@ -71,6 +71,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     try:
         api.ping_anna_thermostat()
     except OSError:
+        _LOGGER.debug("Ping failed, retrying later.", exc_info=True)
```

Comment:
We don't end logging messages with period.

Reply:
Darn, I knew that, should have looked better and not only correct the typo.